src/components/map/WorldMap.tsx

```diff
@@ -0,0 +1,339 @@
+import React, { useRef, useState, useMemo } from "react";
+import { ComposableMap, Geographies, Geography, Graticule, Marker, Sphere, ZoomableGroup, Point as TPoint } from "react-simple-maps";
+import { Tooltip, TooltipRefProps } from "react-tooltip";
+import useConfig from "@hooks/useConfig";
+import useGeographies from "@hooks/useGeographies";
+import { GEO_CENTER_POINTS } from "@constants/mapCentres";
+import { GEO_EU_COUNTRIES } from "@constants/mapEUCountries";
+import { TGeography } from "@types";
+import { LinkWithQuery } from "@components/LinkWithQuery";
+import GeographySelect from "./GeographySelect";
+import { ZoomControls } from "./ZoomControls";
+
+const geoUrl = "/data/map/world-countries-50m.json";
+
+type TSvgGeo = {
+  geometry: { type: string; coordinates: TPoint[] };
+  id: string;
+  properties: { name: string };
+  rsmKey: string;
+  svgPath: string;
+  type: string;
+};
+
+type TGeoFamilyCounts = {
+  UNFCCC: number;
+  EXECUTIVE: number;
+  LEGISLATIVE: number;
+};
+
+type TGeoMarkers = {
+  lawsPolicies: number;
+  unfccc: number;
+};
+
+type TGeographyWithCoords = TGeography & { coords: TPoint; familyCounts: TGeoFamilyCounts; markers: TGeoMarkers };
+
+type TGeographiesWithCoords = { [key: string]: TGeographyWithCoords };
+
+const geoStyle = (isActive: boolean) => {
```

Comment:
Not a blocker ... Just a note to chat about trying to separate style from behavior into separate files so its clear what is changing.